src/schema/v2/searchCriteriaLabel.ts

```diff
@@ -435,3 +446,36 @@ async function getPartnerLabels(partnerIDs: string[], partnerLoader) {
     })
   )
 }
+
+async function getArtistSeriesLabels(
+  artistSeriesIDs: string[],
+  gravityGraphQLLoader
+) {
+  if (!gravityGraphQLLoader || !artistSeriesIDs?.length) return []
+
+  return Promise.all(
+    artistSeriesIDs.map(async (id) => {
+      const data = await gravityGraphQLLoader({
+        query: gql`
+          query GetArtistSeriesLabelsQuery($id: ID!) {
+            artistSeries(id: $id) {
+              internalID
+              title
+            }
+          }
+        `,
+        variables: { id },
+      })
+      const {
+        artistSeries: { internalID, title },
+      } = data
+
+      return {
+        name: "Artist Series",
+        displayValue: title,
+        field: "artistSeriesIDs",
+        value: internalID,
```

Comment:
Supplying `internalID` as the value for the artist series, as opposed to `slug`. (See recent [comment](https://artsyproduct.atlassian.net/browse/ONYX-478?focusedCommentId=59355) and [discussion](https://artsy.slack.com/archives/C05EQL4R5N0/p1697647419307239).)

Reply:
yes, I've drafted a PR (https://github.com/artsy/eigen/pull/9558) on Friday, but tests fail and input name is also not shown.